city_scrapers/spiders/losca_Metro_Transit.py

```diff
@@ -0,0 +1,91 @@
+    timezone = "America/Los_Angeles"
+    start_urls = ["https://metro.legistar.com/Calendar.aspx"]
+
+    custom_settings = {
+        "ROBOTSTXT_OBEY": False,
+    }
+
+    def parse_legistar(self, events):
+        for event in events:
+            start = self.legistar_start(event)
+            if start:
+                meeting = Meeting(
+                    title=event.get("Name", {}).get("label", "No Title"),
+                    description="",
+                    classification=self._parse_classification(event),
+                    start=start,
+                    end=None,
+                    all_day=False,
+                    time_notes="",
+                    location=self._parse_location(event),
+                    links=self._parse_links(event),
+                    source=self.legistar_source(event),
+                )
+
+                meeting["status"] = self._get_status(meeting)
+                meeting["id"] = self._get_id(meeting)
+
+            yield meeting
+
+    def _parse_classification(self, item):
+        """Parse or generate classification from allowed options."""
+        name_label = item.get("Name", {}).get("label", "").lower()
+        if "committee" in name_label:
+            return COMMITTEE
+        if "board" in name_label:
+            return BOARD
+        if "council" in name_label:
+            return CITY_COUNCIL
+        return NOT_CLASSIFIED
+
+    def _parse_links(self, item):
+        """Parse or generate links to additional information."""
+        links = []
+
+        if isinstance(item.get("Name"), dict) and item["Name"].get("url"):
+            links.append(
+                {
+                    "href": item["Name"]["url"],
+                    "title": item["Name"].get("label", "Meeting Details"),
+                }
+            )
+
+        if isinstance(item.get("Agenda"), dict) and item["Agenda"].get("url"):
+            links.append(
+                {
+                    "href": item["Agenda"]["url"],
+                    "title": item["Agenda"].get("label", "Agenda"),
+                }
+            )
+
+        if isinstance(item.get("iCalendar"), dict) and item["iCalendar"].get("url"):
+            links.append({"href": item["iCalendar"]["url"], "title": "iCalendar"})
+
+        if (
+            isinstance(item.get("Audio"), dict)
+            and item["Audio"].get("url")
+            and item["Audio"].get("label") != "Not\xa0available"
+        ):
+            links.append(
+                {"href": item["Audio"]["url"], "title": item["Audio"]["label"]}
+            )
+
+        return links
+
+    def _parse_location(self, item):
+        """Parse or generate location."""
+        if isinstance(item["Meeting Location"], dict):
+            return {
+                "address": item["Meeting Location"]["label"],
+                "name": item["Name"]["label"],
+            }
+        else:
+            return item["Meeting Location"]
```

Comment:
Let's take a sample meeting that has issues with the "location" field:
- On https://metro.legistar.com/calendar.aspx, filter the list with "Service Councils - Meet and Confer" for year 2024, there are 4 meetings shown
- Check the meeting in the first row, with "Meeting Date" on  11/12/2024
![losca_metro_transit_meeting_location](https://github.com/user-attachments/assets/4b69d7ea-a792-42f7-9dd1-8cf9e8070feb)

There are several issues with this function `_parse_location()`:
+ In case `item["Meeting Location"]` is not a dict, it returns a block of text instead of a "dict" of keys "address" and "name". What I got in the file "test_output.csv" is `"1 Gateway Plaza, 3rd Floor Gateway Conference Room, Los Angeles, CA 90012 Hybrid in-person/virtual meeting"`. Correct "location" json-dumped text should be
```
"{'address': '1 Gateway Plaza, 3rd Floor Gateway Conference Room, Los Angeles, CA 90012 Hybrid in-person/virtual meeting', 'name': 'Service Councils - Meet and Confer'}"
```

+ If you take a closer look into the "Meeting Location" column, you'll see that this html part `<em>Hybrid in-person/virtual meeting</em>` is not about the location, it's more about "description". And I also noticed that all meetings have empty "description" extracted. So I suggest to use this `<em>` tag for `Meeting.description`, and also exclude it from `Meeting.location["address"]`.